optaplanner-examples/src/main/resources/org/optaplanner/examples/flightcrewscheduling/solver/flightCrewSchedulingSolverConfig.xml

```diff
@@ -19,9 +19,8 @@
     <unionMoveSelector>
       <changeMoveSelector/>
       <swapMoveSelector/>
-      <!-- TODO Use a sorted pillar to heavily improve feasibility and solution quality https://issues.jboss.org/browse/PLANNER-1217 -->
-      <!--<pillarChangeMoveSelector/>-->
-      <!--<pillarSwapMoveSelector/>-->
+      <pillarChangeMoveSelector/>
+      <pillarSwapMoveSelector/>
```

Comment:
do we want to enable SEQUENTIAL here?
It depends on your benchmark results - if this way is better (I think you showed me that), then let's indeed keep it as this.

Reply:
Your choice. The results are nearly identical - one is better here, the other is better there.
I suspect that, when there are additional constraints, the differences will become more pronounced.